app/services/cloud_foundry.rb

```diff
@@ -151,9 +151,13 @@ def self.generate_app(app_name, app_manifest)
       if app_manifest["qafire"]["buildpack"]
         app["buildpack"] = app_manifest["qafire"]["buildpack"]
       end
-      if app_manifest["qafire"]["memory"]
+
+      if app_manifest["qafire"]["deploy_memory"]
+        app["memory"] = to_megabytes(app_manifest["qafire"]["memory"])
```

Comment:
Should it be `app_manifest["qafire"]["deploy_memory"]`? Maybe there's a way to make a helper function instead of all these if statements that have copy-paste errors?